src/views/Item20Items.js

```diff
@@ -0,0 +1,34 @@
+import kind from '@enact/core/kind';
+import Item from '@enact/sandstone/Item';
+
+const Item20Items = kind({
+	name: 'Item20Items',
+
+	render: () => (
+		<div style={{ height: '700px', overflow: 'auto' }}>
+			<Item id="item">Hello Item</Item>
```

Comment:
Let's use <Group> element. Please find it on Sandstone.


for (let i = 0; i < 50; i++) {
	itemData.push(`Item ${i}`);
}
...

	<Group childComponent={Item}>
						{itemData}
					</Group>